src/pages/PMT_PaymentWizard.page

```diff
@@ -260,7 +260,10 @@ Copyright (c) 2008 Evan Callahan, evanc@npowerseattle.org, NPower Seattle, 403 2
                                     <label class="slds-form-element__label" for="oppAmount4">{!$ObjectType.Opportunity.Fields.Amount.Label}</label>
                                     <div class="slds-form-element__control">
                                         <span class="slds-form-element__static">
-                                            <apex:outputField value="{!currentOpp.Amount}" id="oppAmount4"/>
+                                            <span class="slds-form-element__addon">{!currencySymbol}</span>
```

Comment:
Any idea why these are showing in this diff? Some sort of problem with auto merge from master?

Reply:
This (and a few others) were change I had made to the page way back when I worked on advanced currency management support in rollups. I think at some point when doing a manual merge conflict sync, I overwrote those changes requiring them to be recreated. Here's the full commit: https://github.com/SalesforceFoundation/Cumulus/pull/2950/commits/8a31168af298c4cfb78eea4e11c17b5f72d8d39f